nixos/modules/services/web-apps/filesender.nix

```diff
@@ -0,0 +1,261 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  format = pkgs.formats.php { finalVariable = "$config"; };
+
+  cfg = config.services.filesender;
+  simpleSamlCfg = config.services.simplesamlphp.filesender;
+  fpm = config.services.phpfpm.pools.filesender;
+
+  phpPackage = pkgs.php.withExtensions (
+    { enabled
+    , all
+    }: with all;
+    [
+      xml
+      mbstring
+    ]
+    ++ enabled
+  );
+
+  filesenderConfigPhpFile = format.generate "config.php" cfg.settings;
+
+  filesenderConfigDirectory = pkgs.stdenv.mkDerivation {
+    name = "filesender-config";
+    phases = [ "buildtree" ];
+    buildtree = ''
+      mkdir $out
+      cp ${filesenderConfigPhpFile} $out/config.php
+    '';
+  };
+in
+{
+  meta = {
+    maintainers = with lib.maintainers; [ nhnn ];
+  };
+
+  options.services.filesender = with lib; {
+    enable = mkEnableOption "Filesender";
+    package = mkPackageOption pkgs "filesender" { };
+    user = lib.mkOption {
+      description = ''
+        User under which filesender runs.
+      '';
+      type = lib.types.str;
+      default = "filesender";
+    };
+    database = {
+      createLocally = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Create the PostgreSQL database and database user locally.
+        '';
+      };
+      hostname = mkOption {
+        type = types.str;
+        default = "localhost";
```

Comment:
It is confusing that the default is not a socket path despite it being used.

Reply:
Fixed